### PR TITLE
Feature/687

### PR DESCRIPTION
**Pull request by gattrCh**
**Prerequisites: Always create two feature branches (frontend + backend) even if your feature only affects one of them**

**DEV:**
> Development
- [ ] Acceptance tests by dev [OpenShift feature environment]
  - [ ] Moved to "Ready for review"

> Review
- [ ] Pull-request reviewed + approved (including Code Coverage) by reviewer [GitHub]
- [ ] Inform dev after review through MEGA-PR (by reviewer) [Chat Space]
- [ ] Version upgrade by dev [GitHub Issue Milestone + maven revision]
- [ ] Merge to main by dev [GitHub]
  - [ ]  Re-add SNAPSHOT to version and push on main [maven revision]
  - [ ]  Make sure all feature branches are deleted [frontend + backend]
  - [ ]  Moved to "Ready for deployment"

> Deployment
- [ ] Integration and Unit tests on main environment passed [Workflow]
- [ ] Deployed to the test environment [GitOps Workflow triggered]
  - [ ] Moved to "Ready for testing"

**PO:**
> Testing
- [ ] Acceptance tests on the test enviroment passed
- [ ] Inspected and approved
  - [ ] Moved to "Ready for release"

### CODE REVIEW

File: src/app/domains/project-management/feature-project-management/feature-project-management.component.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@
   align-items: center;
 }
 
+.test {
```
Comment (Ruhsi) [inline src/app/domains/project-management/feature-project-management/feature-project-management.component.scss:22]: test??

File: src/app/domains/project-management/feature-project-management/feature-project-management.component.ts [hunk 2]
```diff
@@ -386,6 +387,8 @@ export class FeatureProjectManagementComponent implements OnInit, OnDestroy {
     return this.pmService.getEntries(this.selectedYear, this.selectedMonth, false);
   }
 
+
```
Comment (Ruhsi) [inline src/app/domains/project-management/feature-project-management/feature-project-management.component.ts:390]: hm 🤔

File: src/app/domains/project-management/ui-common/billable-times/billable-times.component.html [hunk 3]
```diff
@@ -17,5 +17,14 @@
     <td *ngIf="nonBillableTimes === '00:00'" class="times-col-right">
       <mat-icon>horizontal_rule</mat-icon>
     </td>
+    <td class="times-col-center">&nbsp;/&nbsp;</td>
+    <td *ngIf="percentageOfHoursSpentInThisProject !== 0.0"
+        class="times-col-outer-right"
+        matTooltip="Der Mitarbeiter hat {{percentageOfHoursSpentInThisProject}}% seiner Gesamtstunden (dieses Monats), in dieses Projekt investiert.">
+      {{percentageOfHoursSpentInThisProject + "%"}}
```
Comment (Ruhsi) [inline src/app/domains/project-management/ui-common/billable-times/billable-times.component.html:24]: @Ollitod ist hier das % außerhalb schöner/besser? also {{percentageOfHoursSpentInThisProject}} %

**Review by Ruhsi — CHANGES_REQUESTED** [review]

**Review by Ollitod — APPROVED** [review]

File: src/app/domains/project-management/ui-common/billable-times/billable-times.component.html (hunk 3 above)
Comment (Ollitod) [reply src/app/domains/project-management/ui-common/billable-times/billable-times.component.html:24]: Ja, außen finde ich es besser.

**Pull request merged**